modules/nodes-descheduler-profiles.adoc

```diff
@@ -90,9 +90,54 @@ Do not enable `CompactAndScale` with any of the following profiles: `LifecycleAn
 
 endif::nodes[]
 ifdef::virt[]
-Use the `LongLifecycle` profile to enable the descheduler on a virtual machine. This is the only descheduler profile currently available for {VirtProductName}. To ensure proper scheduling, create VMs with CPU and memory requests for the expected load.
+Use the `DevKubeVirtRelieveAndMigrate` or `LongLifecycle` profile to enable the descheduler on a virtual machine.
+
+[IMPORTANT]
+====
+You can not have both `DevKubeVirtRelieveAndMigrate` and `LongLifeCycle` enabled at the same time.
+====
 endif::virt[]
 
+`DevKubeVirtRelieveAndMigrate`:: This profile is an enhanced version of the `LongLifeCycle` profile. It evicts pods from high-cost nodes to reduce overall resource expenses and enable workload migration. Node costs can include the following:
++
+--
+* **Resource utilization**: Increased resource pressure raises the overhead for running applications.
+* **Node maintenance**: A higher number of containers on a node increases resource consumption and maintenance costs.
+
+The profile enables the `LowNodeUtilization` strategy with the `EvictionsInBackground` alpha feature. The profile also exposes the following customization fields:
+
+* `devLowNodeUtilizationThresholds`: Sets experimental thresholds for the LowNodeUtilization strategy.
+* `devActualUtilizationProfile`: Enables load-aware descheduling.
+* `devDeviationThresholds`: Bases thresholds on average utilization instead of fixed values.
+* `devMultiSoftTainting`: Applies multiple soft-taints to nodes instead of a single taint.
```

Comment:
- `devEnableSoftTainter`: Have the operator deploying the soft-tainter component to dynamic set/remove soft taints as a hint for the scheduler based on the same criteria used for the descheduling